mdx-models/src/main/java/com/mx/path/model/mdx/accessor/products/ShoppingCartBaseAccessor.java

```diff
@@ -0,0 +1,81 @@
+package com.mx.path.model.mdx.accessor.products;
+
+import com.mx.path.core.common.accessor.API;
+import com.mx.path.core.common.accessor.AccessorMethodNotImplementedException;
+import com.mx.path.core.common.gateway.GatewayAPI;
+import com.mx.path.core.common.gateway.GatewayClass;
+import com.mx.path.gateway.accessor.Accessor;
+import com.mx.path.gateway.accessor.AccessorConfiguration;
+import com.mx.path.gateway.accessor.AccessorResponse;
+import com.mx.path.model.mdx.model.MdxList;
+import com.mx.path.model.mdx.model.products.Product;
+
+/**
+ * Accessor base for shopping cart operations
+ *
+ * <p>See <a href="https://developer.mx.com/drafts/mdx/product/#shopping-cart">specifications</a>
+ */
+@GatewayClass
+@API(specificationUrl = "https://developer.mx.com/drafts/mdx/product/#shopping-cart")
+public class ShoppingCartBaseAccessor extends Accessor {
+
+  public ShoppingCartBaseAccessor() {
+  }
+
+  /**
+   * @param configuration
+   * @deprecated Use the default constructor, the configuration is set by the accessor construction context code
+   */
+  @Deprecated
```

Comment:
We don't need to implement deprecated methods for a new base accessor.

Reply:
I just heard the shopping cart design was just changed by Matt and Joe. That might affect this MR as well.

Reply:
You'll have to talk to Joe Hansen. I think the new spec just has is_selected field on Product.